cmake_modules/FindPortaudio.cmake

```diff
@@ -39,55 +39,81 @@ else (PORTAUDIO_LIBRARIES AND PORTAUDIO_INCLUDE_DIRS)
     )
     set(PORTAUDIO_FOUND TRUE)
   else (PORTAUDIO2_FOUND)
-    find_path(PORTAUDIO_INCLUDE_DIR
-      NAMES
-        portaudio.h
-      PATHS
-        /usr/include
-        /usr/local/include
-        /opt/local/include
-        /sw/include
-    )
-   
-    find_library(PORTAUDIO_LIBRARY
-      NAMES
-        portaudio
-      PATHS
-        /usr/lib
-        /usr/local/lib
-        /opt/local/lib
-        /sw/lib
-    )
-   
-    find_path(PORTAUDIO_LIBRARY_DIR
-      NAMES
-        portaudio
-      PATHS
-        /usr/lib
-        /usr/local/lib
-        /opt/local/lib
-        /sw/lib
-    )
-   
-    set(PORTAUDIO_INCLUDE_DIRS
-      ${PORTAUDIO_INCLUDE_DIR}
-    )
-    set(PORTAUDIO_LIBRARIES
-      ${PORTAUDIO_LIBRARY}
-    )
-   
-    set(PORTAUDIO_LIBRARY_DIRS
-      ${PORTAUDIO_LIBRARY_DIR}
-    )
-   
-    set(PORTAUDIO_VERSION
-      18
-    )
-   
+
+    if (NOT WIN32)
+      find_path(PORTAUDIO_INCLUDE_DIR
+        NAMES
+          portaudio.h
+        PATHS
+          /usr/include
+          /usr/local/include
+          /opt/local/include
+          /sw/include
+      )
+
+      find_library(PORTAUDIO_LIBRARY
+        NAMES
+          portaudio portaudio_x64
+        PATHS
+          /usr/lib
+          /usr/local/lib
+          /opt/local/lib
+          /sw/lib
+      )
+
+      find_path(PORTAUDIO_LIBRARY_DIR
+        NAMES
+          portaudio
+        PATHS
+          /usr/lib
+          /usr/local/lib
+          /opt/local/lib
+          /sw/lib
+      )
+
+      set(PORTAUDIO_INCLUDE_DIRS
+        ${PORTAUDIO_INCLUDE_DIR}
+      )
+      set(PORTAUDIO_LIBRARIES
+        ${PORTAUDIO_LIBRARY}
+      )
+
+      set(PORTAUDIO_LIBRARY_DIRS
+        ${PORTAUDIO_LIBRARY_DIR}
+      )
+
+      set(PORTAUDIO_VERSION
+        18
+      )
+    else (NOT WIN32)
+
+        find_path(PORTAUDIO_INCLUDE_DIR
+            NAMES portaudio.h
+            PATHS ${CMAKE_SOURCE_DIR}/../portaudio/include
+        )
+        find_library(PORTAUDIO_LIBRARY
+            NAMES portaudio_x64.lib portaudio_x86.lib libportaudio.a
```

Comment:
ah, missed the NOT.

if you want to keep cmake arguments to a minimum, add portaudio to the external libraries and build while building sc.